build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
 
   dependencies {
     classpath libs.plugin.kotlin
-    classpath libs.plugin.android
+    classpath 'com.android.tools.build:gradle:7.3.1'
```

Comment:
Not sure if this change was intented?  `7.4-rc1` is in the toml.
`/paparazzi/build.gradle` and your new compileOnly still uses the 7.4 so you might get overridden.

Reply:
I have a feeling this was him testing the solution.

See the last `classpath` below and https://github.com/cashapp/paparazzi/blob/master/settings.gradle#L9 where that plugin is being replaced.

Reply:
I think I understand this more now. As I ran into an issue with this.

The the paparazzi-gradle-plugin can use the latest version, but for testing and building the paparazzi AAR an older version can be used.

@jrodbx I suggest splitting it into 3:
 * `libs.plugin.android.build` (for AAR build)
 * `libs.plugin.android.test` (for test-fixtures)
 * `libs.plugin.android.api` (for dependency)

The `api` should be behind a bit (stable), the `build` can be the latest alpha, and the `test` is the best compatible version.